pallets/xyk/src/lib.rs

```diff
@@ -57,51 +83,187 @@ decl_module! {
             Ok(())
         }
 
-        fn create_pool(origin,
+        /// Creates liquidity pool pair for Asset 1 and Asset 2
+        fn create_liquidity_pool(origin,
```

Comment:
Liquidity pool should always be created with some initial liquidity amount

Reply:
Makes sense, will call mint function.